src/style.css

```diff
@@ -1,3 +1,73 @@
+@media (min-width: 1024px), (max-width: 1200px) {
+  header {
+    min-width: 924px;
+    max-width: 1100px;
+  }
+}
+
+html {
+  --textcolor: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: auto;
+  padding: 0 50px;
+}
+
+.home_page {
+  height: 40px;
+}
+
+.box {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.nav__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-transform: uppercase;
+  font-family: "Roboto", "SansSerifCollection", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  height: 60px;
+  margin-left: 20px;
+  padding: 0;
+}
+
+.nav__link:first-child {
+  margin: 0;
+}
+
+.nav__link:hover {
+  cursor: pointer;
+  color: var(--textcolor);
+}
+
+.is-active {
+  color: var(--textcolor);
+  padding-top: 23px;
```

Comment:
it looks no good 
Your `is_active` selector should be with position: relative.
Your `is_active:after` selector should be with position: absolute and positioned with top/left/bottom/right rules
```suggestion
```